os/syscall/syscall_lookup.h

```diff
@@ -257,7 +321,32 @@ SYSCALL_LOOKUP(rmdir,                   1, STUB_rmdir)
 SYSCALL_LOOKUP(umount,                  1, STUB_umount)
 SYSCALL_LOOKUP(unlink,                  1, STUB_unlink)
 SYSCALL_LOOKUP(ftruncate,               2, STUB_ftruncate)
+#else
+SYSCALL_LOOKUP(fsync,                   1, STUB_notsupport_int)
+SYSCALL_LOOKUP(mkdir,                   2, STUB_notsupport_int)
+SYSCALL_LOOKUP(mount,                   5, STUB_notsupport_int)
+SYSCALL_LOOKUP(rename,                  2, STUB_notsupport_int)
+SYSCALL_LOOKUP(rmdir,                   1, STUB_notsupport_int)
+SYSCALL_LOOKUP(umount,                  1, STUB_notsupport_int)
+SYSCALL_LOOKUP(unlink,                  1, STUB_notsupport_int)
+SYSCALL_LOOKUP(ftruncate,               2, STUB_notsupport_int)
 #  endif
+#else
```

Comment:
This is the else case of `#if CONFIG_NFILE_DESCRIPTORS > 0`. But it does not have `fsync~ftruncate`.
It will cause syscall mismatch.

As you miss this, this looks much complex.